<launch>

    <include file="$(find sbg_ros_driver-1.0.7)/launch/sbg_ellipse_helios.launch"/>

    <node name="key_teleop" pkg="key_teleop" type="key_teleop.py" output="screen"/>

    <node name="pololu" pkg="BathyBoatNav" type="pololu">
        <param name="Path" 			type="string" 	value="/dev/pololu_servo_serial"/>
        <param name="Cons_channel" 	type="string" 	value="/key_vel"/>
        <param name="Turn_gap" 		type="int" 		value="100"/>
    </node>

 	<node name="rosbag_record_diag" pkg="rosbag" type="record" 
 			args="record -o /home/lallemfa/bags/teleop -a"/>
 			
    <node name="tcp_server_send" pkg="BathyBoatNav" type="tcp_server" args="29200 true" output="screen" />
        
    <node name="tcp_server_recv" pkg="BathyBoatNav" type="tcp_server" args="23200 false" output="screen" />


</launch>